config/initializers/python.rb

```diff
@@ -42,6 +45,12 @@ def process_requirements(requirements_file)
              'Jupyter notebook rendering will be disabled. ' \
              "To enable notebook rendering run: #{pip} install -r #{Rails.root.join('requirements-jupyter.txt')}"
       end
+      unless (qr_requirements - installed).empty?
+        warn 'MARKUS WARNING: not all packages required to process scanned exams ' \
+             'are installed. Exam scanning will not be enabled. ' \
+             'To enable exam scanning  run: ' \
```

Comment:
@pretendWhale Please delete the double space

Reply:
Did this for the `qr_requirements` warning. The `scanner_requirements` warning also has a double space - let me know if you think that's worth taking action on.

Reply:
@pretendWhale good catch, yeah please go ahead and fix it in both

Reply:
@david-yz-liu Done!